lib/policy_machine.rb

```diff
@@ -244,6 +244,15 @@ def batch_find(type:, query: {}, config: {}, &blk)
     end
   end
 
+  def pluck(type:, fields:, options:)
+    if policy_machine_storage_adapter.respond_to?(:pluck)
+      policy_machine_storage_adapter.send(:pluck, {type: type, fields: fields, options: options})
+    else
+      Warn.once("WARNING: 'pluck' is not implemented for storage adapter #{policy_machine_storage_adapter.class}")
+      nil
```

Comment:
I think we should actually hard raise here.

Reply:
Now it throws the error.